.github/workflows/build-tfm.yml

```diff
@@ -0,0 +1,29 @@
+name: Build-TFM
+
+on:
+  push:
+    branches:
+        - '**'
+  pull_request:
+    branches:
+        - '**'
+
+jobs:
+  build-tfm:
+    runs-on: ubuntu-latest
+    container: ghcr.io/armmbed/mbed-os-env:master-latest
+    strategy:
+      matrix:
+        board: [ARM_MUSCA_S1, ARM_MUSCA_B1, NU_M2354]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          apt-get update -y
+          apt-get install -y srecord
+          mbed-tools deploy
+          pip install -r mbed-os/requirements.txt
```

Comment:
The Docker image already has requirements installed, unless we want to be extra sure they are up to date?

Reply:
I mean https://github.com/ARMmbed/mbed-os/blob/50c135292c42b69f7c87dba351af378a9e1e8f77/docker_images/mbed-os-env/Dockerfile#L42-L45

Reply:
Good catch. I'll remove this `pip install` line.